cfgov/jinja2/v1/_includes/organisms/reg-comment.html

```diff
@@ -1,50 +1,95 @@
+   value.id:            (optional) Sets the id attribute in the form's markup.
+                        If not set, the form will be assigned a base id of
+                        `o-reg-comment_` with a random number appended.
 
    ========================================================================== #}
 
-<div class="o-email-signup">
-{% if value.heading %}
-    <h2 class="header-slug">
-        <span class="header-slug_inner">
-            {{ value.heading }}
-        </span>
-    </h2>
-{% endif %}
-
-<form id="{{ value.id or ('o-email-signup_' ~ range(1, 100) | random) }}"
-      action="/subscriptions/new/"
-      method="POST"
-      enctype="application/x-www-form-urlencoded">
-    {% if value.text %}
-        <p>
-            {{ value.text }}
-        </p>
-    {% endif %}
-
-    {% set field = 'email' %}
-    {% for form in value.form_field %}
-        {{ render_stream_child(form) }}
-    {% endfor %}
-
-    <div class="form-group">
-        <input type="hidden" name="code" value="{{ value.gd_code }}">
-    </div>
-</form>
-</div>
+<div class="o-reg-comment">
+    <form id="{{ value.id or ('o-reg-comment_' ~ range(1, 100) | random) }}"
+          action="/regulation-comment/new/"
+          method="POST"
+          enctype="multipart/form-data">
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="general_comment">
+                    Enter your comments
+                </label>
+                <textarea name="general_comment" rows="5" required></textarea>
+
+                <p class="u-mb30">
+                    If you would like to include an attachment with your response,
+                    please submit your comment at
+                    <a class="icon-link icon-link__external-link"
+                       href="https://www.regulations.gov/">
+                        <span class="icon-link_text">Regulations.gov</span>
+                    </a>.
+                </p>
+            </div>
+        </div>
+
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="first_name">
+                    First name
+                </label>
+                <input type="text" id="first_name" name="first_name" required>
+            </div>
+        </div>
+
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="last_name">
+                    Last name
+                </label>
+                <input type="text" id="last_name" name="last_name" required>
+            </div>
+        </div>
 
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="email">
+                    Email address
+                    <span class="label_note">(optional)</span>
+                </label>
+                <input type="text" id="email" name="email">
```

Comment:
input type should be `email` and you probably want placeholder text with the value set to `example.com`